examples/thermodynamics/determination_of_constants_in_van_der_waals_equation.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+from sympy import solve, Symbol, Eq
+from symplyphysics import print_expression, Quantity, units, convert_to
+from symplyphysics.laws.thermodynamics.equations_of_state import van_der_waals_state_equation as van_der_waals_law
+
+# Example from 2.24 in https://studfile.net/preview/1772224/page:8/
+
+temperature_before = Symbol("temperature_before")
+temperature_after = Symbol("temperature_after")
+volume = Symbol("volume")
+amount_of_substance = Symbol("amount_of_substance")
+pressure_before = Symbol("pressure_before")
+pressure_after = Symbol("pressure_after")
+
+parameter_a = Symbol("parameter_a")
+parameter_b = Symbol("parameter_b")
+
+state_equation_before = van_der_waals_law.law.subs({
+    van_der_waals_law.volume: volume,
+    van_der_waals_law.temperature: temperature_before,
+    van_der_waals_law.pressure: pressure_before,
+    van_der_waals_law.bonding_forces_parameter: parameter_a,
+    van_der_waals_law.molecules_volume_parameter: parameter_b,
+    van_der_waals_law.amount_of_substance: amount_of_substance,
+    van_der_waals_law.units.molar_gas_constant: Quantity(8.31446262 * units.joules / (units.moles * units.kelvins))
```

Comment:
It already contains `units.molar_gas_constant` which is a proper quantity. You do not need to define your own.